src/Literate.jl

```diff
@@ -936,15 +936,24 @@ function execute_block(sb::Module, block::String; inputfile::String, fake_source
         end
     end
     popdisplay(disp) # IOCapture.capture has a try-catch so should always end up here
-    if c.error && !continue_on_error
-        error("""
-             $(sprint(showerror, c.value))
-             when executing the following code block from inputfile `$(Base.contractuser(inputfile))`
-
-             ```julia
-             $block
-             ```
-             """)
+    if c.error
+        if continue_on_error
+            err = c.value
+            if err isa LoadError # include_string may wrap error in LoadError
+                err = err.error
+            end
+            error_output = "ERROR: " * sprint(showerror, err)
+            return nothing, error_output, disp.data
```

Comment:
Probably this should be appended to `c.output` in case there are stderr/stdin output before the block throws an error.